projects/rocprofiler-systems/docker/Dockerfile.ubuntu

```
@@ -23,7 +23,8 @@ RUN apt-get update && \
     apt-get install -y apt-utils autoconf autotools-dev bash-completion bison \
         build-essential chrpath cmake curl flex gettext git-core gnupg2 iproute2 \
         libnuma1 libopenmpi-dev libpapi-dev libpfm4-dev librpm-dev libsqlite3-dev \
-        libtool libudev1 lsb-release m4 ninja-build python3-pip rpm texinfo wget && \
+        libtool libudev1 lsb-release m4 ninja-build python3-pip rpm texinfo wget \
+        nlohmann-json3-dev && \
```

```suggestion
        libtool libudev1 lsb-release m4 ninja-build nlohmann-json3-dev python3-pip \
        rpm texinfo wget && \
```
Sorting list alphabetically.